### ISSUE TITLE
I'd like to be able to save a default sort order for when you search or are in a Browse Category

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/pomegranate#737]

**Comment by hackmastera** [comment]
related to #697

**Comment by kelea99** [comment]
@hackartisan and @tpendragon . perhaps an easier way forward, that would empower users to create order themselves on pages via a widget (the item row widget, specifically).  If we can constrain proportions of the thumbnail to help standardize the size, I have a feeling that may be an easier and more sustainable route.  We could also workshop the size of the thumbnail, thereby also being able to similarly address and close #858.  

There is also the option of getting the item embed widget to function properly. But I would love your thoughts on this.

**Comment by hackartisan** [comment]
That might make sense for the use case of a guided exhibit experience -- there wouldn't be too many items. Is the idea that it might be preferred to showcase them in that way?

**Comment by kelea99** [comment]
@hackartisan Yup! As luck would have it This idea was a direct result of a recent use case. An individual from Mudd is creating an online companion to a physical exhibit that was in Mudd previously. This is how they would like to display the items
